org-formation/700-aws-sso/_tasks.yaml

```diff
@@ -356,6 +356,26 @@ SsoViewerPlus:
       - arn:aws:iam::aws:policy/AWSElasticBeanstalkReadOnly
       - arn:aws:iam::aws:policy/CloudWatchLogsReadOnlyAccess
       - arn:aws:iam::aws:policy/AWSLambda_ReadOnlyAccess
+    inlinePolicy: >-
```

Comment:
is there not an AWS managed policy for this?

Also why not just allow viewer role this access?

Reply:
No, AWS doesn't provide a managed policy, the most they offer is guidance on building your own: https://docs.aws.amazon.com/awsaccountbilling/latest/aboutv2/billing-example-policies.html 

And to be honest, I'm not exactly sure what the use-cases are for viewer vs viewer-plus, but I was presuming that viewer-plus was intended for any use-case that requires more than the standard ViewOnlyAccess policy. Does it make more sense to add this inlinePolicy to both viewer and viewer-plus?

Reply:
don't forget about the list of `managedPolicies` for this role.  when you assign a user this role they will get all that access.  This means that if your intention is to give a user in finance only `cost explorer` access you are also giving them access to all the `managedPolicies`.

if we instead put this on viewer role then they will only get access to cost explore and  `arn:aws:iam::aws:policy/job-function/ViewOnlyAccess` policy.

which should it be?  i'm not sure myself.

Reply:
Can we simply create a new role (separate from Viewer and Viewer-Plus), meant for people who need to look at Cost Explorer?

Reply:
Since one of our goals is to allow any cloud user to track their own costs, I think I'm going to add the inline policy to both the viewer and viewer plus roles so that anyone with AWS access also has cost explorer access (scoped to the member account the IAM user is logged in to).

Reply:
@brucehoff your idea is indeed possible and i think a valid alternative solution. 

@jesusaurus what about the `developers` and `auditor` role? shouldn't those roles get access to as well?  If you are going to add duplicate inline policies to multiple roles I think it would be better to create a custom managed policy that you can then add to each role's `managedPolicies` list.